src/BSON/Decimal128.stub.php

```diff
@@ -9,7 +9,7 @@
 
 final class Decimal128 implements Decimal128Interface, \JsonSerializable, Type, \Serializable
 {
-    final public function __construct(string $value = '') {}
```

Comment:
I take it this was always inconsistent, as the constructor has always required 1 parameter and uses `Z_PARAM_STRING`.

Reply:
That is correct - this may be due to me being lazy and using PHPStorm's stubs as base for these and subsequently forgetting to double-check this method: https://github.com/JetBrains/phpstorm-stubs/blob/9d134fcc6c8d4cbb8ffd8e65e25a49fea18a7a0d/mongodb/mongodb.php#L2015